style.css

```diff
@@ -0,0 +1,461 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.youtube-logo {
+    width: 10%;
+}
+
+i {
+    font-size: 150%;
+}
+
+#nav-bar{
+    height: 56px;
+    display: flex;
+    box-shadow: 0px 0px 0px 3px rgba(226, 226, 226, 0.25);
+}
+
+#nav-bar > ul {
+    list-style: none;
+    margin: 0;
```

Comment:
this navbar should have some margin around it so your logos don't get floated to the extreme ends.